app/controllers/users/registrations_controller.rb

```diff
@@ -9,7 +9,7 @@ def new
       invite_code = params[:invite_code]
       invitation = invitation(invite_code)
       if invitation
-        @user = User.new(email: invitation.email, cohort: invitation.cohort)
+        @user = User.new(email: invitation.email, cohort_id: invitation.cohort_id)
```

Comment:
@srt32 I'm curious about the change here. Is this a style preference, does it provide an improvement in the lookup or something else entirely? Thanks!

Reply:
Great question! The change is needed because `Cohort` is no longer an ActiveRecord model but a PORO that is masquerading as a model. Therefore, `User#belongs_to :cohort` doesn't work anymore but `user.cohort_id` and `user.cohort_id=` does work. 

We could update `user` to have a `cohort=` method, which under the hood would do `cohort_id=` and we could leave this interface the same.  I decided to minimize the amount of methods we needed to map and opted to update how we call it here. I'm open to revisiting that decision though if it would make things less confusing.

happy to talk more IRL if I could explain better.